istio-proxy/istio-proxy.spec

```diff
@@ -52,11 +51,16 @@ BuildRequires:  platform-python-devel
 
 Source0:        proxy-%{git_commit}.tar.gz
 
+ExclusiveArch:  x86_64
+
 %description
 The Istio Proxy is a microservice proxy that can be used on the client and server side, and forms a microservice mesh. The Proxy supports a large number of features.
 
 %prep
 %setup -q -n proxy-%{git_commit}
+%setup -q -D -T -n proxy-%{git_commit} -a 10
```

Comment:
is this right? What's with `-a 10`?

Reply:
Not actually sure what this is doing, but this is how we are doing it in product. This PR just synchronizes the two. Let me look into that.

Reply:
Option -a (which stands for after) expands those sources after entering. Their arguments are source numbers from the spec file preamble.

From: https://rpm-packaging-guide.github.io/

Reply:
GN was labeled as source 10. Based on that, I think this was expanding that.